service/dependencies.gradle

```diff
@@ -41,7 +41,7 @@ dependencies {
   implementation group: "bio.terra", name: "terra-resource-buffer-client", version: "0.198.42-SNAPSHOT"
 
   // Cloud Resource Library
-  implementation group: 'bio.terra', name: 'terra-cloud-resource-lib', version: "1.2.30-SNAPSHOT"
+  implementation group: 'bio.terra', name: 'terra-cloud-resource-lib', version: "1.2.30-rt-bump-SNAPSHOT"
```

Comment:
TODO: update when https://github.com/DataBiosphere/terra-cloud-resource-lib/pull/182 is merged